ros_gz_sim_demos/worlds/ros_gz.sdf

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" ?>
+<!--
+  ROS 2 <-> Gazebo Sim plugin demo.
+
+-->
+<sdf version="1.8">
+  <world name="demo">
+
+    <physics name="1ms" type="ignored">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+    </physics>
+    <plugin
+      filename="ignition-gazebo-physics-system"
+      name="gz::sim::systems::Physics">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-user-commands-system"
+      name="gz::sim::systems::UserCommands">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-scene-broadcaster-system"
+      name="gz::sim::systems::SceneBroadcaster">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-contact-system"
+      name="gz::sim::systems::Contact">
```

Comment:
Use `gz-sim` instead of `ignition-gazebo`.

Reply:
Changed in https://github.com/gazebosim/ros_gz/pull/511